lib/mongo/mongoClient.js

```diff
@@ -110,14 +110,15 @@ module.exports = function(config){
       }
       requestCount = 0;
 
-      mongodb.MongoClient.connect(config.connectionString, function(err, db){
-        if (db != null) {
+      mongodb.MongoClient.connect(config.connectionString, { useNewUrlParser: true }, function(err, client){
+        if (client != null) {
           if (mongoClient != null) {
-            db.close();
+            client.close();
             return;
           }
           started = true;
-          mongoClient = db;
+          db = client.db("data_test");
```

Comment:
I'm guessing that we either want to extract this the `connectionString` (see [`gatekeeper`'s PR](https://github.com/tidepool-org/gatekeeper/pull/21/files#diff-7fa343e057026503d750bfb872a400beR114)). I also would think that we want the default to be `data`. `data_test` would make sense for the unit tests.